chapter_03_abstractions.asciidoc

```diff
@@ -127,7 +127,7 @@ Now we need to write the interesting business logic. When we have to tackle a
 problem from first principles, we usually try to write a simple implementation,
 and then refactor towards better design. We'll use this approach throughout the
 book, because it's how we write code in the real world: start with a solution
-to the smallest part of the problem, and then iteratively make the solution
+to the smallest part of the problem, and then interactively make the solution
```

Comment:
Why this? I _definitely_ meant iteratively.

Reply:
no idea. maybe i fat-fingered it?  will put it back.  thanks for the :eyes: